src/classes/STG_PanelRD_CTRL.cls

```diff
@@ -13,79 +13,69 @@
-    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT 
-    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN 
-    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
     POSSIBILITY OF SUCH DAMAGE.
 */
 /**
 * @author Salesforce.org
 * @date 2014
 * @group Settings
-* @description The controller for the Recurring Donations panel 
+* @description The controller for the Recurring Donations panel
 */
-public with sharing class STG_PanelRD_CTRL extends STG_Panel { 
+public with sharing class STG_PanelRD_CTRL extends STG_Panel {
 
     /*********************************************************************************************************
-    * @description Returns the string Id of the Recurring Donations panel. 
+    * @description Returns the string Id of the Recurring Donations panel.
     */
     public override string idPanel() { return 'idPanelRD'; }
- 
+
     /*********************************************************************************************************
     * @description holds value to evaluate change in forecast months
     */
     private integer originalForecastMonths;
-    
+
     /*********************************************************************************************************
     * @description The panel's constructor
     */
     public STG_PanelRD_CTRL() { }
-    
+
     /*********************************************************************************************************
-    * @description Initiliazes the classes data 
+    * @description Initiliazes the classes data
     */
     public void init(){
         originalForecastMonths = (integer)STG_Panel.stgService.stgRD.npe03__Opportunity_Forecast_Months__c;
     }
-    
+
     /*********************************************************************************************************
-    * @description Action Method to save RD settings, deal with a change in forecast months, 
+    * @description Action Method to save RD settings, deal with a change in forecast months,
     * and schedule the nightly job if appropriate.
     * @return null
     */
-    public override PageReference saveSettings() { 
+    public override PageReference saveSettings() {
```

Comment:
@force2b , can you please double check changes (especially the `STG_PanelRD_CTRL` class) since it was a team work with Meena?